specifyweb/specify/migrations/0002_geo.py

```diff
@@ -98,24 +109,33 @@ def create_default_collection_types(apps, schema_editor):
+                ('decimal1', models.DecimalField(blank=True, db_column='Decimal1', decimal_places=10, max_digits=22, null=True)),
+                ('decimal2', models.DecimalField(blank=True, db_column='Decimal2', decimal_places=10, max_digits=22, null=True)),
+                ('decimal3', models.DecimalField(blank=True, db_column='Decimal3', decimal_places=10, max_digits=22, null=True)),
                 ('text1', models.TextField(blank=True, db_column='Text1', null=True)),
                 ('text2', models.TextField(blank=True, db_column='Text2', null=True)),
                 ('text3', models.TextField(blank=True, db_column='Text3', null=True)),
-                ('collection', models.ForeignKey(db_column='CollectionID', on_delete=specifyweb.specify.models.protect_with_blockers, related_name='collectionobjectgroups', to='specify.collection')),
-                ('createdbyagent', models.ForeignKey(db_column='CreatedByAgentID', null=True, on_delete=specifyweb.specify.models.protect_with_blockers, related_name='+', to='specify.agent')),
-                ('modifiedbyagent', models.ForeignKey(db_column='ModifiedByAgentID', null=True, on_delete=specifyweb.specify.models.protect_with_blockers, related_name='+', to='specify.agent')),
+                ('yesno1', models.BooleanField(blank=True, db_column='YesNo1', null=True)),
+                ('yesno2', models.BooleanField(blank=True, db_column='YesNo2', null=True)),
+                ('yesno3', models.BooleanField(blank=True, db_column='YesNo3', null=True)),
+                ('version', models.IntegerField(blank=True, db_column='Version', default=0, null=True)),
+                ('timestampcreated', models.DateTimeField(db_column='TimestampCreated', default=django.utils.timezone.now)),
+                ('timestampmodified', models.DateTimeField(blank=True, db_column='TimestampModified', default=django.utils.timezone.now, null=True)),
+                ('collection', models.ForeignKey(db_column='CollectionID', on_delete=protect_with_blockers, related_name='collectionobjectgroups', to='specify.collection')),
+                ('cogtype', models.ForeignKey(db_column='COGTypeID', on_delete=protect_with_blockers, related_name='collectionobjectgroups', to='specify.collectionobjectgrouptype')),
+                ('createdbyagent', models.ForeignKey(db_column='CreatedByAgentID', null=True, on_delete=protect_with_blockers, related_name='+', to='specify.agent')),
+                ('modifiedbyagent', models.ForeignKey(db_column='ModifiedByAgentID', null=True, on_delete=protect_with_blockers, related_name='+', to='specify.agent')),
             ],
             options={
                 'db_table': 'collectionobjectgroup',
                 'ordering': (),
             },
         ),
         migrations.CreateModel(
-            name='CollectionObjectGroupJoin',
+            name='CollectionObjectGroupJoin', # add as dependant to collection object group
```

Comment:
Is this a TODO comment? 
Make sure you remove this if completed, finish it before merging, or provide a more descriptive comment (ideally linking to an Issue/PR/Commit, etc.)